Unity.Mathematics.TestProject/Packages/manifest.json

```diff
@@ -1,12 +1,13 @@
 {
   "dependencies": {
+    "com.unity.burst": "1.2.3",
     "com.unity.ide.rider": "1.1.4",
     "com.unity.ide.vscode": "1.1.4",
     "com.unity.mathematics": "file:../../src",
     "com.unity.package-validation-suite": "0.8.1-preview",
     "com.unity.properties": "0.10.4-preview",
     "com.unity.test-framework": "1.1.11",
-    "com.unity.modules.androidjni": "1.0.0"
```

Comment:
Is this intentional?

Reply:
Yes, in the fact that I removed it.  The test project has no need to call into android code at this point in time.